devel/bazel/Portfile

```diff
@@ -29,115 +29,66 @@ set extra_args ""
+                    sha256  820a94dbb14071ed6d8c266cf0c080ecb265a5eea65307579489c4662c2d582a \
+                    size    235201891
 
     set bazel_min_xcode 12.0
     compiler.blacklist-append {clang < 1200}
     compiler.blacklist-append {macports-clang-[5-9].0}
-    set min_darwin    16
-
 } elseif { "bazel-3.7" eq ${subport} } {
+    PortGroup       obsolete 1.0
 
     # Specific version for py-tensorflow - DO NOT CHANGE
-    github.setup      bazelbuild ${name} 3.7.2
-    revision          1
-
-    # livecheck none is correct here, see above.
-    livecheck.type    none
-
-    checksums         rmd160  abf23b277194854bbd4e4390ab6b3f971d26826c \
-                      sha256  de255bb42163a915312df9f4b86e5b874b46d9e8d4b72604b5123c3a845ed9b1 \
-                      size    313677756
-
-    compiler.blacklist-append {clang < 710}
-    compiler.blacklist-append {macports-clang-[7-9].0}
-    set min_darwin    16
+    github.setup    bazelbuild ${name} 3.7.2
+    revision        1
 
+    # Obsolete Date: 2023-08-01
+    replaced_by     bazel
 } elseif { "bazel-3.5" eq ${subport} } {
+    PortGroup       obsolete 1.0
 
     # Specific version for py-tensorflow - DO NOT CHANGE
-    github.setup      bazelbuild ${name} 3.5.1
-    revision          1
-
-    # livecheck none is correct here, see above.
-    livecheck.type    none
-
-    checksums         rmd160  8c52fab5214d862b83459f807c39ae392db49a75 \
-                      sha256  67eae714578b22d24192b0eb3a2d35b07578bbd57a33c50f1e74f8acd6378b3c \
-                      size    307982168
-
-    compiler.blacklist-append {clang < 710}
-    compiler.blacklist-append {macports-clang-[7-9].0}
-    set min_darwin    16
-    supported_archs   x86_64
-
-    # https://trac.macports.org/ticket/62551
-    set extra_args "--host_javabase=@local_jdk//:jdk"
+    github.setup    bazelbuild ${name} 3.5.1
+    revision        1
 
+    # Obsolete Date: 2023-08-01
+    replaced_by     bazel
 } elseif { "bazel-3.1" eq ${subport} } {
+    PortGroup       obsolete 1.0
 
     # Specific version for py-tensorflow - DO NOT CHANGE
-    github.setup      bazelbuild ${name} 3.1.0
-    revision          1
-
-    # livecheck none is correct here, see above.
-    livecheck.type    none
-
-    checksums         rmd160  b3238d0539e757a2a7b07f2e4475f1cefdc6307c \
-                      sha256  d7f40d0cac95a06cea6cb5b7f7769085257caebc3ee84269dd9298da760d5615 \
-                      size    269384742
-
-    compiler.blacklist-append {clang < 710}
-    compiler.blacklist-append {macports-clang-[7-9].0}
-    set min_darwin    16
-    supported_archs   x86_64
-
-    # https://trac.macports.org/ticket/62551
-    set extra_args "--host_javabase=@local_jdk//:jdk"
+    github.setup    bazelbuild ${name} 3.1.0
+    revision        1
 
+    # Obsolete Date: 2023-08-01
+    replaced_by     bazel
 } elseif { "bazel-2.0" eq ${subport} } {
+    PortGroup       obsolete 1.0
     
```

Comment:
Based on a quick lint check, it looks like you have trailing whitespace on line 76.

Reply:
Thanks, fixed.